<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QWidget" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>在线课堂</string>
  </property>
  <widget class="QLabel" name="Label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>192</width>
     <height>62</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:36pt;&quot;&gt;在线课堂&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ClassRoomIdEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>5edd4083-d20d-4b75-9981-d11188b20a60</string>
   </property>
   <property name="placeholderText">
    <string>请输入您的教室号</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="UidEdit">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>190</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>fenglin</string>
   </property>
   <property name="placeholderText">
    <string>请输入您的用户名</string>
   </property>
  </widget>
  <widget class="QComboBox" name="BizTypeBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>260</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>大班课</string>
    </property>
   </item>
  </widget>
  <widget class="QRadioButton" name="TeacherButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>老师端</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QRadioButton" name="StudentButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>89</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>学生端</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cover">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>251</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Users/fenglin/Desktop/123.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="EnterRoomButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>380</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(36, 156, 211);
font: 9pt &quot;微软雅黑&quot;;
color:rgb(255, 255, 255)</string>
   </property>
   <property name="text">
    <string>进 入 教 室</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
